docs/realm.js

```diff
@@ -331,7 +342,10 @@ Realm.defaultPath;
  *        The purpose of open_ssl_verify_callback is to enable custom certificate handling and to solve cases where
  *        OpenSSL erroneously rejects valid certificates possibly because OpenSSL doesn't have access to the
  *        proper trust certificates.
- *
+ *   - `partial` - Whether this Realm should be opened in 'partial synchronization' mode.
+ *        Partial synchronization mode means that no objects are synchronized from the remote Realm
```

Comment:
How about reversing the explanation? Instead of "no objects, except", then "only those matching...". e.g.
"Partial synchronisation only synchronizes those objects that match the query specified in contrast to the normal mode of operation that synchronises all objects in a remote Realm."